src/main/java/com/jcabi/github/mock/MkIssueLabels.java

```diff
@@ -182,4 +217,15 @@ private String xpath() {
         );
     }
 
+    /**
+     * Returns a set of all of the issue's labels.
+     * @return Set of label names
+     */
+    private Set<String> labelSet() {
```

Comment:
I get you there, but we aren't using any operations based on the `Set` contract right now, so better to use the most basic interface that we need.